lib/perfdata/influxdbcommonwriter.hpp

```diff
@@ -78,7 +79,7 @@ void InfluxdbCommonWriter::StatsFunc(const Dictionary::Ptr& status, const Array:
 	for (const typename InfluxWriter::Ptr& influxwriter : ConfigType::GetObjectsByType<InfluxWriter>()) {
 		size_t workQueueItems = influxwriter->m_WorkQueue.GetLength();
 		double workQueueItemRate = influxwriter->m_WorkQueue.GetTaskCount(60) / 60.0;
-		size_t dataBufferItems = influxwriter->m_DataBuffer.size();
+		size_t dataBufferItems = influxwriter->m_DataBufferSize;
```

Comment:
Btw. could this also be done by a high-prio WQ item?

Reply:
Could be implemented that way but it would risk having to wait for at least the currently-running work queue job. This might be a flush, which does a TLS connection and might run into timeouts and so on.